{
    "id": "CVE-2023-41105",
    "product": "n/a",
    "version": "n/a",
    "vulnerability": [
        "n/a"
    ],
    "description": "An issue was discovered in Python 3.11 through 3.11.4. If a path containing '\\0' bytes is passed to os.path.normpath(), the path will be truncated unexpectedly at the first '\\0' byte. There are plausible cases in which an application would have rejected a filename for security reasons in Python 3.10.x or earlier, but that filename is no longer rejected in Python 3.11.x.",
    "poc": {
        "reference": [
            "No PoCs from references."
        ],
        "github": [
            "https://github.com/JawadPy/CVE-2023-41105-Exploit",
            "https://github.com/fkie-cad/nvd-json-data-feeds",
            "https://github.com/nomi-sec/PoC-in-GitHub",
            "https://github.com/toxyl/lscve"
        ]
    }
}